Week3/homework/js-exercises/4-shopping.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+let groceryItemArray = ["Milk", "Banana"];
```

Comment:
`const` instead of `let` please because if you check again, you aren't reassigning this variable anywhere, right? 😉